Mixpanel/MPUITableViewBinding.m

```diff
@@ -122,4 +122,18 @@ - (UITableView *)parentTableView:(UIView *)cell {
     return nil; // this view is not within a tableView
 }
 
+- (BOOL)isEqual:(id)other {
```

Comment:
Yeah, I think it is alright to remove it. I am just concerned if for instance we have 2 Event Bindings, and one if a TableViewBinding and the other is a ControlViewBinding and they have the same name and path, we could get an equals because we aren't checking if both have the class MPUITableViewBinding. But I think it is impossible to have them be the same path? What do you think?